tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -27,20 +27,20 @@ Feature: restrict Sharing
     And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt" in the server
     And user "Brian" has logged in using the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with users in their groups
     Given the setting "shareapi_only_share_with_group_members" of app "core" has been set to "yes" in the server
+#   shares only with groups members is not available now in the oCIS webUI
     When the user opens the share dialog for folder "simple-folder" using the webUI
-
     And the user types "Ali" in the share-with-field
     Then "user" "Alice Hansen" should be listed in the autocomplete list on the webUI
     But "user" "Alison Cooper" should not be listed in the autocomplete list on the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with groups they are member of
     Given the setting "shareapi_only_share_with_membership_groups" of app "core" has been set to "yes" in the server
     When the user opens the share dialog for folder "simple-folder" using the webUI
-
+#   shares only with groups members is not available now in the oCIS webUI
```

Comment:
similar here above the scenario and maybe this will be clearer?
```suggestion
#   Configuration for sharing only within the group member is not available in ocis and won't be implemented in the same way as OC10 see issue https://github.com/owncloud/ocis/issues/1328 for more detail
```

Reply:
I put this comment in the wrong place in this scenario.
About this comment  `#   shares only with groups members is not available now in the oCIS webUI` i was just following the existing pattern `# Reshare permission is not available now in the oCIS webUI`

Reply:
there is an related issue tag **@issue-ocis-1317** at the top of the feature file so I didn't mention it in the comment.I hope this works